README.md

```diff
@@ -3,5 +3,3 @@
 [![Travis CI status](https://api.travis-ci.org/ascoderu/ascoderu.github.io.svg?branch=development)](https://travis-ci.org/ascoderu/ascoderu.github.io)
 
 Public information site for Ascoderu, hosted via Github-Pages and Cloudflare at [ascoderu.ca](https://ascoderu.ca).
-
-Built with [Bootstrap](http://getbootstrap.com/docs/3.3/) and [FullPage](https://alvarotrigo.com/fullPage/).
```

Comment:
You should remove only the part of FullPage since we still use Bootstrap

Reply:
Done in ef5b029.